app/views/assignments/show.html.erb

```diff
@@ -0,0 +1,22 @@
+<p id="notice"><%= notice %></p>
```

Comment:
Ah low key this may not be necessary if yall are converting all this to react. Double check if we can delete some of these html.erb files or we keeping them for react components.

Reply:
assigment/show, assigment/new, assigment/edit... etc.

Reply:
Yeah I think they're able to get to these pages via the admin dashboard so these pages are just to keep those alive. I don't think we have any plans to style these though so they'll just be plain